protocol/0012-POSR-position_resolution.md

```diff
@@ -22,147 +22,48 @@ Any trader that has insufficient collateral to cover their margin liability is r
+- `full disposal size` (e.g. `20`): once net absolute value of network's open volume is at or below that value, the network will attempt to dispose the remaining amount in one go,
+- `max fraction of book side within liquidity bounds consumed` (e.g. `m=0.05`): once the network chooses the size of its order (`s_candidate`) the effective size will be calcualted as `s_effective=min(m*N, s_candidate)`, where `N` is the sum of volume (on the side of the book with which the network's order will be matching) that falls within the range implied by the `market.liquidity.priceRange` [parameter](./0044-LIME-lp_mechanics.md#market-parameters),
+- `try on next block if trade unsuccessful` (e.g. `true`): if after the network forms and submits its IOC order the resulting trade volume is `0` it will try to reform and resubmit the order in subsequent blocks until it successfully achieves a trade of any size, once the next disposal time step anniversary is reached no further attempts are made and process starts from scratch.
 
-CloseOutTrade3 {
-  buyer: Network,
-  seller: Trader3,
-  size: 2,
-  price: 113.33,
-  type: 'safety-provision'
-}
-```
-
-This results in the open position sizes for all distressed traders and the network entities to be zero.
-
-`// OpenPosition of Network =  -3 +5 -4 +2 = 0`
-`// OpenPosition of Trader1 =  +5 -5 = 0`
-`// OpenPosition of Trader2 = -4 +4 =  0`
-`// OpenPosition of Trader3 =  +2 - 2 = 0`
-
-#### STEP 5
-
-The collateral from distressed traders is moved to the insurance pool
-
-```json
-// sent by Settlement Engine to the Collateral Engine
-TransferRequest1 {
-  from: [Trader1_MarginAccount],
-  to: MarketInsuranceAccount,
-  amount: Trader1_MarginAccount.size, // this needs to be the full amount
-}
-
-TransferRequest2 {
-  from: [Trader2_MarginAccount],
-  to: MarketInsuranceAccount,
-  amount: Trader2_MarginAccount.size, // this needs to be the full amount
-}
-
-TransferRequest3 {
-  from: [Trader3_MarginAccount],
-  to:  MarketInsuranceAccount,
-  amount: Trader3_MarginAccount.size, // this needs to be the full amount
-}
-```
-
-#### STEP 6
-
-Traders from step 3 need to be settled.
-
-Prior to STEP 3 trades, assume Trader 4 and Trader 5 had the following open positions.
-
-`// OpenPosition of Trader4 =  -3`
-`// OpenPosition of Trader5 =  15`
-
-Trader 4 has therefore closed out 2 contracts through the `LiquiditySourcingTrade` 1. These need to be settled against the trade price.
-
-```json
-TransferRequest4 {
-  from: [MarketInsuranceAccount],
-  to:  Trader4_MarginAccount,
-  amount: (120 - PreviousMarkPrice) * -2, // this is the movement since the last settlement multiplied by the volume of the closed out amount
-}
-
-```
+Assume the price range implied by the `market.liquidity.priceRange` is `[a, b]`. Once the network has worked out a size of its immediate or cancel limit order it sets it's price to `a` if it's a sell order or `b` if it's a buy order, and it submits the order.
```

Comment:
Why not a market order?

Reply:
Just thought that would reduce the scope for manipulation of a network party or for it to make non-sensical trades.

Reply:
So liquidation strategy will be part of the market framework. Where should these paramters be specified, what validation should we have in place, etc...? Esentially: we need to update the 0001-MKTF-market_framework spec file for this I think